renderdoc/driver/vulkan/wrappers/vk_get_funcs.cpp

```diff
@@ -1167,3 +1167,28 @@ VkResult WrappedVulkan::vkGetDeferredOperationResultKHR(VkDevice device,
 {
   return ObjDisp(device)->GetDeferredOperationResultKHR(Unwrap(device), operation);
 }
+
+void WrappedVulkan::vkGetAccelerationStructureBuildSizesKHR(
+    VkDevice device, VkAccelerationStructureBuildTypeKHR buildType,
+    const VkAccelerationStructureBuildGeometryInfoKHR *pBuildInfo,
+    const uint32_t *pMaxPrimitiveCounts, VkAccelerationStructureBuildSizesInfoKHR *pSizeInfo)
+{
+  ObjDisp(device)->GetAccelerationStructureBuildSizesKHR(Unwrap(device), buildType, pBuildInfo,
```

Comment:
This struct `VkAccelerationStructureBuildGeometryInfoKHR` contains acceleration structures which need to be unwrapped before being passed down.